Documentation/configuration.md

```diff
@@ -1,8 +1,8 @@
 # rkt configuration
 
-`rkt` reads configuration from two directories - a **system directory** and a **local directory**.
-The system directory defaults to `/usr/lib/rkt`, and the local directory to `/etc/rkt`.
-Both locations can be changed with command line flags.
+`rkt` reads configuration from two or three directories - a **system directory**, a **local directory** and, if provided, a **user directory**.
+The system directory defaults to `/usr/lib/rkt`, the local directory to `/etc/rkt`, and the user directory to empty string.
```

Comment:
English-wise it should be "an empty string". But this doesn't tell us what it means. So +1 to Alban's comments.